assets/js/webPushMain.js

```diff
@@ -0,0 +1,147 @@
+/*** Web push test ****/
+var applicationServerPublicKey = 'BFtBVunx6Lg_4ziA6b_Oe-9iqQetudiBzkukb1UOOWItZlbdPXCnMLjTfM6gnbmzrusaGlifwWWaFRWnef_H40Y';
+// Public Key
+// BFtBVunx6Lg_4ziA6b_Oe-9iqQetudiBzkukb1UOOWItZlbdPXCnMLjTfM6gnbmzrusaGlifwWWaFRWnef_H40Y
+// Private Key
+// F1eZLlbqpH5d8nHetf6xzzBnlgq8WbFlQ2A9jEl9X4E
```

Comment:
Is it fine including this in a public repository? (if not, please consider revoke this key soon)

Reply:
This is only to easily test, not sensitive information at all. In the real life yes we would not put this information here.